### DISCUSSION TITLE
Is it possible to keep a popover aligned with an anchor that moves?

### DISCUSSION THREAD

**Discussion by rynobax** [radix-ui/primitives#2006] [Help] [answered]
The popover stays aligned when I scroll a container that the popover anchor is in, but when the container's size changes it does stay aligned.  Is there a way to achieve that behavior?

Example:
https://codesandbox.io/s/moving-popover-example-v7urer?file=/src/App.tsx

**Comment by joaom00** [comment]
I'm afraid it's not possible

**Reply by benoitgrelard** [reply]
I think that's because floating-ui is being efficient by default and only re-adjusting based on certain events.
There is a mode to force it on a `requestAnimationFrame` basis (more expensive) so technically it would be possible to expose this in one way or another.

**Reply by linkb15** [reply]
@benoitgrelard is it possible that to have this method you mentioned be exposed and can be used to trigger the update manually (like by clicking a button for example)?

**Reply by benoitgrelard** [reply]
We'll see if we can expose a way to do this.

**Comment by benoitgrelard** [comment, marked as answer]
This is fixed by #2092
